README.md

```diff
@@ -10,8 +13,29 @@ Join the Vaadin Fusion community chat in https://vaad.in/fusion
 
 **Note**: Currently the code of Vaadin Fusion is hosted at https://github.com/vaadin/flow together with the Flow framework. The Fusion-only code will be ported to this repository in the near future. Tickets can already be created in this repository.
 
-## Packages
+## TypeScript
+
+The collection of frontend and NodeJS utilities used by Fusion.
+
+| Package                 | Status |
+|-------------------------|--------|
+| [@vaadin/common-frontend](./ts/common-frontend) | [![Latest Stable Version](https://img.shields.io/npm/v/@vaadin/common-frontend.svg)](https://www.npmjs.com/package/@vaadin/common-frontend) |
```

Comment:
I guess this should be `ts/form` instead of `ts/common-frontend` ?

`common-frontend` package is in a [different repo](https://github.com/vaadin/flow-fusion-common/tree/main/frontend/packages/common-frontend)

Reply:
I know this was wrong already before moving this content but might as well fix it while moving.

Reply:
sure